assets/queries/dockerCompose/privileged_containers_enabled/query.rego

```diff
@@ -0,0 +1,18 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	resource := input.document[i]
+	service_parameters := resource.services[name]
+    privileged := service_parameters[name2].privileged
```

Comment:
Please, do deeper research about where the field `privileged` can be defined. I think this approach is not correct.

Observe the output of the docker-compose, please:

![image](https://user-images.githubusercontent.com/74001161/160177499-c4ed093f-fe6a-4866-8d43-9a236aaea530.png)


Probably, the best approach is the following (Please, confirm it):

```suggestion
    privileged := service_parameters.privileged
```

![image](https://user-images.githubusercontent.com/74001161/160177769-7fc833b6-2234-4d3e-9d2c-2520264286fb.png)